tailwind.config.ts

```diff
@@ -154,7 +154,10 @@ const config = {
           high: "hsla(var(--background-high))",
         },
 
-        // TODO: Add border color tokens to match DS
+        // TODO: Add all border color tokens to match DS
+        border: {
+          DEFAULT: "hsla(var(--border))",
+        },
```

Comment:
Curious about this, @nloureiro do we have a default color for our borders?

Reply:
this was added for the homepage and we should use on other pages too
![Screen Shot 2024-09-27 08 41 20 AM](https://github.com/user-attachments/assets/a7b8c931-cc7e-4c15-a4af-5fd1d16de5ed)

Reply:
![Screen Shot 2024-09-27 08 42 18 AM](https://github.com/user-attachments/assets/5d2cf1cc-3e7e-4e44-9d18-664fd2433cd5)